staging/src/k8s.io/api/go.mod

```diff
@@ -16,6 +16,7 @@ replace (
 	github.com/imdario/mergo => github.com/imdario/mergo v0.3.5
 	github.com/mattn/go-colorable => github.com/mattn/go-colorable v0.0.9
 	github.com/onsi/ginkgo => github.com/openshift/ginkgo v4.7.0-origin.0+incompatible
+	github.com/openshift/build-machinery-go => github.com/openshift/build-machinery-go v0.0.0-20210806203541-4ea9b6da3a37
```

Comment:
This is a little surprising.  I don't know whether it's wrong though.  Did `go mod vendor` add this?

Reply:
I'm not sure, to be hoenst. There is a rather complicated script that gets run as part of the bump process. I executed the following:

1. Update the replace for `openshift/api`
2. `go mod vendor && go mod tidy`
3. `hack/update-vendor.sh`